src/test/policyestimator_tests.cpp

```diff
@@ -93,24 +87,19 @@ BOOST_AUTO_TEST_CASE(BlockPolicyEstimates)
     }
 
     std::vector<CAmount> origFeeEst;
-    std::vector<double> origPriEst;
     // Highest feerate is 10*baseRate and gets in all blocks,
     // second highest feerate is 9*baseRate and gets in 9/10 blocks = 90%,
     // third highest feerate is 8*base rate, and gets in 8/10 blocks = 80%,
     // so estimateFee(1) should return 9*baseRate.
     // Third highest feerate has 90% chance of being included by 2 blocks,
     // so estimateFee(2) should return 8*baseRate etc...
-    for (int i = 1; i < 10;i++) {
+    for (int i = 1; i < 9;i++) {
```

Comment:
Why is this range decreased?

Reply:
it shouldn't have been decreased, thanks for the catch. see previous comment for the actual place where range should have been decreased.